admin/check/check_config_inc.php

```diff
@@ -95,6 +95,22 @@
 	array( false => 'Issues moved may end up with invalid category id.' )
 );
 
+$t_field_options = array(
+	'bug_report_page_fields',
+	'bug_view_page_fields',
+	'bug_update_page_fields'
+);
+
+foreach( $t_field_options as $t_field_option ) {
+	$t_fields = config_get_global( $t_field_option );
```

Comment:
This will not capture user or global config in the database.  Note that this is not a global config option (it can be in the database as well).  Anyways, not a big deal, but you may want to config changing this to `config_get()` instead of `config_get_global()`.

Reply:
> This will not capture user or global config in the database

Right, I mentioned that already in my note.

> you may want to config changing this to `config_get()` instead of `config_get_global()`

This is not the solution for it, as the check must not depend on currrent project or current user.

I changed to  `config_get( $t_field_option, null, ALL_USERS, ALL_PROJECTS );`, so the case  where the option is set in config_inc.php and the case where the options is set for all projects and all users in database is covered.

The case where someone sets for a specific project and/or a specific user is still not covered.